tests/integration/tier1/convert-offline-systems/prepare_system.py

```diff
@@ -0,0 +1,33 @@
+from envparse import env
+
+
+def test_prepare_system(shell):
+    assert shell("yum install dnsmasq bind-utils wget -y").returncode == 0
+
+    # Install katello package
+    sat_url = "dogfood.sat.engineering.redhat.com"
+    pkg_url = "https://dogfood.sat.engineering.redhat.com/pub/katello-ca-consumer-latest.noarch.rpm"
+    pkg_dst = "/usr/share/convert2rhel/subscription-manager/katello-ca-consumer-latest.noarch.rpm"
+    assert shell("wget --no-check-certificate --output-document {} {}".format(pkg_dst, pkg_url)).returncode == 0
+    assert shell("rpm -i {}".format(pkg_dst)).returncode == 0
+
+    # Replace urls in rhsm.conf file to the dogfood satellite server
+    assert shell("sed -i 's|hostname = .*|hostname = {}|' /etc/rhsm/rhsm.conf".format(sat_url)).returncode == 0
+    assert (
+        shell("sed -i 's|baseurl = .*|baseurl = https://{}/pulp/repos|' /etc/rhsm/rhsm.conf".format(sat_url)).returncode
+        == 0
+    )
```

Comment:
What about doing the replacement in python instead of calling some bash commands?
It would be:
- more readable
- easier for debugging if something will go wrong
- unified with code bellow (edit of `/etc/dnsmasq.conf`)

Reply:
Thanks for the idea. I rewrite it in python and put it inside a function.